src/js/components/productList/ProductListViewSettings.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function ProductListViewSettings() {
+    return (
+        <div className="ProductListViewSettings">
```

Comment:
fajnie gdybyś zdążyła zrobić state w tym komponencie i możliwość zaznaczania wybranej wartości - czy to ilości produktów do wyświetlania czy ilości produktów w wierszu